rollup.config.gl-style-format.ts

```diff
@@ -0,0 +1,44 @@
+import replace from '@rollup/plugin-replace';
```

Comment:
Consider using a single config to all 3 scripts and a single build command in package.json file.